portfolio/src/main/webapp/style.css

```diff
@@ -1,7 +1,18 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+	text-align: center;
```

Comment:
generally 2-space indents are used at google. Overall, you should try to stick with the original styling guidelines when possible (which is another argument for keeping this as 2 space-indents).